.pipelines/scripts/run-e2e.sh

```diff
@@ -31,8 +31,35 @@ get_random_location() {
   echo "${LOCATIONS[${RANDOM} % ${#LOCATIONS[@]}]}"
 }
 
+collect_log() {
+  mkdir "${ARTIFACT_DIR}/kube-system"
+  mkdir "${ARTIFACT_DIR}/node-log"
+  ALL_PODS=($(kubectl get pod -A -owide | grep "kube-system" | awk '{printf("%s ",$2)}'))
+  for POD in "${ALL_PODS[@]}"; do
+      kubectl logs ${POD} -n kube-system > "${ARTIFACT_DIR}/kube-system/${POD}.log"
+  done
+  ALL_NODES=($(kubectl get node | grep "aks-" | awk '{printf("%s ",$1)}'))
+  LOG_IMAGE="mcr.microsoft.com/dotnet/runtime-deps:6.0"
+  for NODE in "${ALL_NODES[@]}"; do
+      NODE_LOG_DIR="${ARTIFACT_DIR}/node-log/${NODE}"
+      mkdir "${NODE_LOG_DIR}"
+      kubectl debug node/${NODE} -it --image=${LOG_IMAGE} -- /bin/sh -c "chroot /host journalctl --no-pager --output=short-precise" > "${NODE_LOG_DIR}/journal.log"
```

Comment:
is it possible just using one script to collect all those logs?

Reply:
> is it possible just using one script to collect all those logs?

Updated.
I don't find some convenient ways to compact the "kubectl debug" instructions. But I try to use arrays to make the code more concise. PTAL.

Reply:
thanks, I think that's more clear.

Reply:
> thanks, I think that's more clear.

Thanks a lot for review!